src/font/font_face.rs

```diff
@@ -44,6 +44,33 @@ impl FontFace {
         font_face
     }
 
+    // Safety: the FT_Face must be valid and not be freed until the `FontFace` is dropped.
```

Comment:
Maybe better to use a doc comment here, no?

Reply:
I will have a look at this tomorrow.